pages/edit_news_page.py

```
@@ -63,3 +81,53 @@ def click_edit_news_btn(self):
+
+    @allure.step("Check comment text")
+    def check_comment_text(self):
+        comment_text = self.get_wait(60).until(EC.presence_of_element_located(self.NEWS_COMMENT))
+        actual_text = comment_text.text
+        assert actual_text == COMMENT_TEXT
+
+    @allure.step("Check commenter name")
+    def check_commenter_name(self):
+        commenter = self.get_wait(60).until(EC.presence_of_element_located(self.COMMENTER))
+        actual_text = commenter.text
+        assert actual_text == self.EXPECTED_USERNAME
```

Ruff (0.12.2)

Use of `assert` detected

(S101)